mlrun/api/runtime_handlers/base.py

```diff
@@ -212,6 +212,7 @@ def monitor_runs(self, db: DBInterface, db_session: Session):
                 logger.warning(
                     "Failed monitoring runtime resource. Continuing",
                     runtime_resource_name=runtime_resource["metadata"]["name"],
+                    project=project,
```

Comment:
Log name only

Reply:
It is the project name but I agree that the variable name is confusing
I tried to change it but it's rooted deeply in the runtime handlers code so I'll leave if for another day..